Pods/Pods.xcodeproj/xcuserdata/mondale.xcuserdatad/xcschemes/Pods-Twitter-Sentiment.xcscheme

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
since you've implemented a networking manager, the pods the network manager is replacing can be removed